var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -795,3 +806,24 @@ def get_llvm_targets_to_build(spec):
         llvm_targets.add("PowerPC")
 
     return list(llvm_targets)
+
+
+def get_llvm_experimental_targets_to_build(spec):
+    targets = spec.variants['experimental_targets'].value
+
+    # Convert targets variant values to CMake LLVM_EXPERIMENTAL_TARGETS_TO_BUILD array.
+    spack_to_cmake = {
```

Comment:
can you stick to a single `spack_to_cmake` map? that'd be easier in case an experimental target becomes a normal target later on.